core/domain/classifier_services.py

```diff
@@ -359,9 +458,8 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
-                                               state_name):
-    """Gets the classifier object from the exploration attributes.
+def get_classifier_training_job(exp_id, exp_version, state_name):
```

Comment:
@seanlip My bad. I was confused about something else in this. I've made this a get_multi() method. PTAL